src/changelog/v0.30.md

```diff
@@ -1,3 +1,24 @@
+## 0.30.3
```

Comment:
These entries should be removed from `unreleased.md`, right?

Reply:
Yes, they should.